Updating uninstallation suggestion (#13279)
DataDog/integrations-core: docs/dev/oauth_for_data_integrations.md

As a part of your pull request, update your README file with an **uninstallation** section under `## Setup` that includes the following instructions (along with any custom instructions you would like to add):

> - Once this integration has been uninstalled, any previous authorizations will be revoked. 

Reviewer: ```suggestion
- Once this integration has been uninstalled, any previous authorizations are revoked. 
```
